### DISCUSSION TITLE
Welcome to windows-activator Discussions!

### DISCUSSION THREAD

**Discussion by iqbalhasandev** [iqbalhasandev/windows-activator#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shazeedul** [comment]
How does it work? Bro.

**Reply by iqbalhasandev** [reply]
Copy All text and then open your windows CMD run as administrator  then paste all code and press enter button ... and wait some moment to fix your activation .

**Comment by arifulhoque7** [comment]
Is it still working???

**Reply by iqbalhasandev** [reply]
yes It's still working .. do you fetch any issue ???

**Comment by Wajeeh03** [comment]
Thanks Man
It Worked

**Reply by iqbalhasandev** [reply]
you are welcome
